ddl/ddl_worker.go

```diff
@@ -278,8 +278,8 @@ func (d *ddl) runDDLJob(t *meta.Meta, job *model.Job) (ver int64) {
 		ver, err = d.onCreateTable(t, job)
 	case model.ActionDropTable:
 		ver, err = d.onDropTable(t, job)
-	case model.ActionAddColumn:
-		ver, err = d.onAddColumn(t, job)
+	case model.ActionAddColumns:
```

Comment:
we must still support `model.ActionAddColumn`, for old DDL jobs before TiDB upgrade.